src/containers/Tasks/Tasks.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+// import axios from '../../axios-tasks';
+import { connect } from 'react-redux';
+import * as actionTypes from '../../store/actions/index'
+import { Redirect } from 'react-router-dom';
+
+import Task from './Task/Task';
+
+// import '../../assets/semantic/semantic.min.css';
+// import { Dimmer, Loader, Image, Segment } from 'semantic-ui-react';
+
+
+class Tasks extends Component {
+    state = {
+        fetchedTasks: [],
+        loading: false
+    };
+
+    componentDidMount = () => {
+        console.log('componentDidMount');
+        if(!this.props.taskList.length){
+            this.props.onInitTasks(this.props.token);
+            this.setState({ loading: true });
+        }
+    };
+
+    deleteTaskHandler = (taskId) => {
+        console.log(taskId);
+    };
+
+    render() {
```

Comment:
Todo Tasks needs to be created/edited/lists as per user basis

for new user created all todos are shown which is wrong